### DISCUSSION TITLE
front

### DISCUSSION THREAD

**Discussion by AkakievKD** [kubescape/kubescape#1746] [General]
Is UI for kubescape already ?

**Comment by matthyx** [comment]
No, but you can check https://artifacthub.io/packages/headlamp/kubescape-plugin/headlamp_kubescape

**Reply by AkakievKD** [reply]
why dont you support extension for lens?

**Reply by matthyx** [reply]
we do, but it hasn't been updated lately https://github.com/kubescape/lens-extension

**Reply by AkakievKD** [reply]
why?

**Reply by dwertent** [reply]
Unfortunately, we do not have the capacity to maintain the Lens extension
